_pytest/tmpdir.py

```diff
@@ -54,9 +54,8 @@ def getbasetemp(self):
             else:
                 # use a sub-directory in the temproot to speed-up
                 # make_numbered_dir() call
-                import getpass
                 temproot = py.path.local.get_temproot()
-                rootdir = temproot.join('pytest-of-%s' % getpass.getuser())
+                rootdir = temproot.join('pytest-of-%s' % get_user())
                 rootdir.ensure(dir=1)
```

Comment:
If two processes from different accounts concurrently run without a USERNAME setting then this ensure or the subsequent make_numbered_dir calls can run into permission problems, no?